source/overview/corba.rst

```diff
@@ -0,0 +1,22 @@
+10 things you should know about CORBA
+=====================================
+
+1.  **You don’t need to know CORBA to work with TANGO**
+
+2.  CORBA is the acronym for **C**\ ommon **O**\ bject **R**\ equest **B**\ roker **A**\ rchitecture and it is a standard defined by the\ ` <https://en.wikipedia.org/wiki/Object_Management_Group>`__\ Object Management Group (OMG)
```

Comment:
This is how to create a link with text other than URL:
```
`Object Management Group (OMG) <https://en.wikipedia.org/wiki/Object_Management_Group>`_
```
@Matteo04052017, please correct that as well.